scripts/tts_dataset_files/cmudict-0.7b_nv22.02

```diff
@@ -133889,6 +133889,7 @@ CSP  S IY1 EH1 S P IY1
 CTR  S IH1 T IY1 AA1 R
 CUDA  K UW1 D AH0
 CUDNN  K UW1 D IY1 EH2 N EH2 N
+CUSTOMIZABLE  K AH1 S T AH0 M AY1 Z UH0 B AH0 L
```

Comment:
shouldn't it be `K **AH2** S T AH0 M AY1 Z **AH0** B AH0 L`?

Reply:
reference: http://www.speech.cs.cmu.edu/tools/lextool.html

Reply:
Updated